awx/playbooks/check_isolated.yml

```diff
@@ -20,12 +20,48 @@
         mode: pull
         delete: yes
         recursive: yes
+      when: ansible_kubectl_host is not defined
 
     - name: Copy daemon log from the isolated host
       synchronize:
         src: "{{src}}/daemon.log"
         dest: "{{src}}/daemon.log"
         mode: pull
+      when: ansible_kubectl_host is not defined
+
+    - name: Copy artifacts from the isolated host.
+      synchronize:
```

Comment:
Although this output doesn't get _directly_ captured, it can end up in log files; we should be good citizens and include ansible `no_log: True`.